### ISSUE TITLE
Sticky library dox

### ISSUE THREAD

**Issue by twiginteractive** [elementor/elementor#7712]
## Prerequisites
- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [ ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
I am looking for some documentation on the modified Sticky library that you guys use. I want to 'stick' an element that doesn't have such an option thru the UI; and I see the library and can inspect it; I would love to know a few of the events/methods so that I can use to workaround the UI limitation.

I can't use a different Sticky library (like Sticky.js) in tandem because i noticed conflicts with the Elementor version.

Any help would be gratefully received - thank you.

**Comment by shilo-ey** [comment]
Hi @twiginteractive 

If I understand correctly what are you referring to- you are trying to add `sticky` property to an element which doesn't support it using Elementor's native interface.

We use our own custom made JS sticky library, which is based on jQuery so if you are familliar with it you will manage to understand how to use it.

The path for the library is `/wp-content/plugins/elementor-pro/assets/lib/sticky/`

Good luck!

**Comment by twiginteractive** [comment]
Heh thanks @shilo-ey  - I've inspected that library and am guessing how to apply it; it would just be nice if there were API dox :)

Like: what is the process Elementor uses in the backend when you select a widget as sticky? It seems to apply a data-settings JSON option that contains the settings (like whether to stick on mobile etc.) then presumably some global JS runs on all elements with that data setting.

Basically any guidance would cut down on the inspection / testing work I have to do :)
